### PR TITLE
Fix typos in docs

### PR DESCRIPTION
**Pull request by chausner**
This fixes various typos and tweaks punctation/grammar in the documentation.

### CODE REVIEW

File: docs/finance-charts/defaults.fsx
Comment (kMutagene) [inline docs/finance-charts/defaults.fsx:1]: I think this file ended up here by mistake, as it already exists in the `general` folder.

**Review by kMutagene — CHANGES_REQUESTED** [review]
Thanks for this thorough check. I think there is a an unintentional file duplication, otherwise LGTM!

File: docs/finance-charts/defaults.fsx
Comment (chausner) [reply docs/finance-charts/defaults.fsx:1]: Oops, not sure how this happened. I removed the copy again.

**Review by kMutagene — APPROVED** [review]

**Comment by kMutagene** [comment]
Thanks!

**Pull request merged**